src/support/SupportResourceListItem.ts

```diff
@@ -0,0 +1,63 @@
+import { Clickable } from '@terra-ui-packages/components';
+import * as qs from 'qs';
+import { useState } from 'react';
+import { div, h, span } from 'react-hyperscript-helpers';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import * as Style from 'src/libs/style';
+import { SupportResourceType } from 'src/support/SupportResourceType';
+
+const listItemStyle = (selected, hovered) => {
+  const style = {
+    ...Style.navList.itemContainer(selected),
+    ...Style.navList.item(selected),
+    ...(selected ? { backgroundColor: colors.dark(0.1) } : {}),
+    paddingLeft: '2rem',
+  };
+  if (hovered) {
+    return {
+      ...style,
+      ...Style.navList.itemHover(selected),
+    };
+  }
+  return style;
+};
+
+export interface SupportResourceListItemProps {
+  resourceType: SupportResourceType;
+  isActive: boolean;
+}
+
+export const SupportResourceListItem = (props: SupportResourceListItemProps) => {
+  const [hovered, setHovered] = useState<boolean>();
```

Comment:
Suggesting adding an initial state so that `hovered` is typed `boolean` instead of `boolean | undefined`.

```suggestion
  const [hovered, setHovered] = useState<boolean>(false);
```